app/src/main/java/org/schabi/newpipe/settings/VideoAudioSettingsFragment.java

```diff
@@ -21,6 +24,17 @@
 public class VideoAudioSettingsFragment extends BasePreferenceFragment {
     private SharedPreferences.OnSharedPreferenceChangeListener listener;
 
+    @Override
+    public void onViewCreated(@NonNull final View rootView,
+                              @Nullable final Bundle savedInstanceState) {
+        super.onViewCreated(rootView, savedInstanceState);
+        findPreference(getString(R.string.switch_gesture_sides_key))
+            .setEnabled(getPreferenceManager().getSharedPreferences()
+                    .getBoolean(getString(R.string.volume_gesture_control_key), true)
+                    && getPreferenceManager().getSharedPreferences()
+                    .getBoolean(getString(R.string.brightness_gesture_control_key), true));
```

Comment:
I think you can replace this custom code with two preference dependencies in the XML file using `android:dependency`.

Reply:
I don't think that this works because you can only set one `android:dependency` in the XML file.